### PR TITLE
Fix #242 - Add template field to CustomForm model

### LINKED ISSUE #242: Can't login to Report site
ksandstrom@mercycorps.org I get an indicators_indicator.country_id error when I try and login to the tola-reports sites. 

     # - I get an indicators_indicator.country_id error when I try and login to the tola-reports sites. 

<br><br>

### PR DESCRIPTION
**Pull request by yilmazbekdemir**
## Purpose
 User wants to create templates from a form. So, they can share this templates with their organizations and create a new from.

## Approach
is_template field added to CutomForm model in order to seperate templates from Forms.

_Related ticket: Humanitec/ActivityAPI#242

### CODE REVIEW

**Comment by jefmoura** [comment]
@yilmazbekdemir 
- You should rebase instead of merging. You need to remove the commit `
@jefmoura
Merge pull request #1092 from jefmoura/234-add-indicator-disaggregati`
- You mustn't change a migration script manually, only in some specific cases like issues with `UUID` and when you need to run a custom script.

File: formlibrary/migrations/0008_customform_is_template.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('formlibrary', '0007_customform_status'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='customform',
+            name='is_template',
+            field=models.BooleanField(default=0),
```
Comment (Menda) [inline formlibrary/migrations/0008_customform_is_template.py]: It is more common to use `False` instead of `0`

File: formlibrary/models.py [hunk 2]
```diff
@@ -215,6 +215,7 @@ class CustomForm(models.Model):
     create_date = models.DateTimeField(null=True, blank=True)
     edit_date = models.DateTimeField(null=True, blank=True)
     created_by = models.ForeignKey('auth.User', related_name='customforms', null=True, blank=True, on_delete=models.SET_NULL)
+    is_template = models.BooleanField(default=0)
```
Comment (Menda) [inline formlibrary/models.py]: It is more common to use False instead of 0

**Comment by yilmazbekdemir** [comment]
`0` used for other models, also for is_public and default_global fields in this CustomForm model. I didn't want to change structure.

File: formlibrary/models.py (hunk 2 above)
Comment (yilmazbekdemir) [reply formlibrary/models.py]: 0 used for other models, also for is_public and default_global fields in this CustomForm model. I didn't want to change structure.

Comment (Menda) [reply formlibrary/models.py]: Fair enough

**Pull request merged**